DataManagementSystem/Client/FileCatalogClientCLI.py

```diff
@@ -412,25 +428,60 @@ def removeFile(self,args):
     except Exception as x:
       print("Error: rm failed with exception: ", x)
       
-  def removeDirectory(self,args):
-    """ Remove file from the catalog
-    """  
-    
-    path = args[0]
-    lfn = self.getPath(path)
-    print("lfn:", lfn)
+  def removeDirectory(self,lfn, recursive=False, forceNonEmpty=False):
+    """ Remove a given directory from the catalog. Remove multiple directories
+        recursively if recursive flag is True. Remove contained files if forceNonEmpty
+        flag is true.
+    """
+
+    if recursive or forceNonEmpty:
+      resultListDirectory = self.fc.listDirectory(lfn, False)
+      if not resultListDirectory['OK']:
+        print('Failed to look up the directory contents')
+        return S_ERROR('Failed to look up the directory contents')
+
+      # Remove subdirectories first
+      dirDict = resultListDirectory['Value']['Successful'][lfn]['SubDirs']
+      dirList = dirDict.keys()
+      if dirList:
+        if recursive:
+          for dirLfn in dirList:
+            result = self.removeDirectory(dirLfn, recursive, forceNonEmpty)
+            if not result['OK']:
+              print('Error: failed to remove directory', dirLfn)
+              return S_ERROR('Failed to remove directory')
+        else:
+          print('Error: failed to remove non empty directory')
+          return S_ERROR("Failed to remove non empty directory")
```

Comment:
This CLI never returns `S_OK/S_ERROR`, simply `return`. What's better?

Reply:
This is not a CLI command function, its output is checked